api/types.go

```diff
@@ -573,6 +587,21 @@ func (po *PinOptions) FromQuery(q url.Values) error {
 		po.UserAllocations = StringsToPeers(strings.Split(allocs, ","))
 	}
 
+	if v := q.Get("expire-at"); v != "" {
+		var tm time.Time
+		err := tm.UnmarshalText([]byte(v))
+		if err != nil {
+			return fmt.Errorf("parameter expire-at invalid")
+		}
+		po.ExpireAt = tm
```

Comment:
if `tm` is after Time.Now() print a warning.

Reply:
tm (expiry time) is supposed to be after time.Now(). Did you mean before time.Now?

Reply:
yes, sorry, before